jekyll/_cci2/collect-test-data.md

```diff
@@ -476,6 +476,29 @@ For more details on `--runInBand`, refer to the [Jest CLI](https://facebook.gith
 
 To access test metadata for a run from the API, refer to the [test-metadata API documentation]( {{ site.baseurl }}/api/v1-reference/#test-metadata).
 
+## Storing Test Coverage Reports
+
+If you would like to upload code coverage reports, or other non-junit
+test-results, consider using a `store-artifacts` step. In the example below,
+Jest is used with the `--collectCoverage` flag enabled, which can then upload
```

Comment:
Should this be part of the Jest section above? instead of a section ## below it?